project.clj

```diff
@@ -6,17 +6,17 @@
   :url "https://github.com/clojure-emacs/refactor-nrepl"
   :license {:name "Eclipse Public License"
             :url "https://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[nrepl "0.9.0"]
-                 ^:inline-dep [compliment "0.3.12"]
-                 ^:inline-dep [http-kit "2.5.3"]
-                 ^:inline-dep [org.clojure/data.json "2.3.1"]
-                 ^:inline-dep [org.clojure/tools.analyzer.jvm "1.2.2"]
+  :dependencies [[nrepl "1.0.0"]
+                 ^:inline-dep [compliment "0.4.0"]
+                 ^:inline-dep [http-kit "2.5.0"]
+                 ^:inline-dep [org.clojure/data.json "2.4.0"]
+                 ^:inline-dep [org.clojure/tools.analyzer.jvm "1.2.3"]
                  ^:inline-dep [org.clojure/tools.namespace "1.1.0" :exclusions [org.clojure/tools.reader]]
                  ^:inline-dep [org.clojure/tools.reader "1.3.6"]
-                 ^:inline-dep [cider/orchard "0.9.1"]
-                 ^:inline-dep [cljfmt "0.8.0" :exclusions [rewrite-clj rewrite-cljs]]
-                 ^:inline-dep [clj-commons/fs "1.6.310"]
-                 ^:inline-dep [rewrite-clj "1.0.699-alpha"]
+                 ^:inline-dep [cider/orchard "0.12.0"]
+                 ^:inline-dep [cljfmt "0.9.2" :exclusions [rewrite-clj rewrite-cljs]]
+                 ^:inline-dep [clj-commons/fs"1.6.310"]
```

Comment:
The tests run fine and dependency is handled correctly, but maybe since the PR is still open, add a space between the `clj-commons/fs` & `"1.6.310"`?